Notify/handler.ts

```diff
@@ -115,13 +117,30 @@ const prepareNotification = (
 > =>
   pipe(
     canSendVerboseNotification(retrieveUserSession, fiscal_code),
-    TE.orElse(_err => TE.of(false)),
+    TE.orElse(_err => {
+      logger.info(`Error retrieving user session, switch to anonymous`);
```

Comment:
Could be useful to promote this log into a ai custom event?

Reply:
Same suggestion here :)

Reply:
Done in [6f46130](https://github.com/pagopa/io-functions-service-messages/pull/21/commits/6f461306eac724a7caa59e6d51b43bbce639089f)